src/components/la-gaufre/LaGaufreV2.stories.tsx

```diff
@@ -0,0 +1,32 @@
+import type { Meta, StoryObj } from "@storybook/react";
+
+import { LaGaufre } from "./LaGaufre";
+import { LaGaufreV2 } from "./LaGaufreV2";
+
+const meta = {
+  title: "Components/LaGaufreV2",
+  component: LaGaufreV2,
+  tags: ["autodocs"],
+  parameters: {
+    layout: "fullscreen",
+  },
+} satisfies Meta<typeof LaGaufre>;
+
+export default meta;
+type Story = StoryObj<typeof meta>;
+
+export const Default: Story = {
```

Comment:
Main generated "Docs" is empty:  
<img width="1162" height="914" alt="Capture d’écran 2025-11-25 à 16 43 50" src="https://github.com/user-attachments/assets/e7bc3ad0-a6c5-4569-bb80-e10a5104b61d" />